wcfsetup/install/files/lib/acp/form/ApplicationManagementForm.class.php

```diff
@@ -0,0 +1,237 @@
+<?php
+
+namespace wcf\acp\form;
+
+use Stripe\Exception\PermissionException;
+use wcf\data\application\ViewableApplicationList;
+use wcf\data\page\Page;
+use wcf\data\page\PageList;
+use wcf\data\page\PageNodeTree;
+use wcf\form\AbstractForm;
+use wcf\system\application\ApplicationHandler;
+use wcf\system\cache\builder\ApplicationCacheBuilder;
+use wcf\system\cache\builder\RoutingCacheBuilder;
+use wcf\system\exception\UserInputException;
+use wcf\system\Regex;
+use wcf\system\WCF;
+use wcf\util\ArrayUtil;
+use wcf\util\FileUtil;
+use wcf\util\StringUtil;
+
+/**
+ * Shows the application management form.
+ *
+ * @author Alexander Ebert
+ * @copyright 2001-2021 WoltLab GmbH
+ * @license GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
+ * @package WoltLabSuite\Core\Acp\Form
+ */
+class ApplicationManagementForm extends AbstractForm
```

Comment:
```suggestion
final class ApplicationManagementForm extends AbstractForm
```
Make everything final by default.